app/joinCircle.js

```diff
@@ -50,78 +54,78 @@ export default function Page() {
 	}
 
 	return (
-		<StyledSafeArea className='bg-offblack  flex-1'>
+		<StyledSafeArea
+			className='bg-offblack flex-1 h-screen'
+			style={{ paddingTop: topInset }}
+		>
 			<KeyboardAwareScrollView bounces={false}>
-				<StyledView className='flex items-center justify-center text-center w-screen h-[90px]'>
-					<StyledText className='text-offwhite font-bold text-4xl'>
-						Join New Circle:
-					</StyledText>
-				</StyledView>
-				<StyledView className='flex-row justify-center'>
-					<StyledView className='w-[300px] h-[300px] border-2 border-offwhite'>
-						<BarCodeScanner
-							mirrorImage={true}
-							fixOrientation={true}
-							className='w-full h-full'
-							ratio='1:1'
-							onBarCodeScanned={
-								scanned ? undefined : handleBarCodeScanned
-							}
-						/>
+				<StyledView className='flex-1 items-center h-screen'>
+					<StyledView className='flex items-center justify-center text-center w-screen h-[90px]'>
+						<StyledText className='text-offwhite font-bold text-4xl'>
+							Join New Circle
+						</StyledText>
 					</StyledView>
+
+					<StyledView className='absolute top-[15%] border-[10px] border-offwhite rounded-xl'>
```

Comment:
The percentages for padding here are causing issues with how the page renders, at least on my device. It makes the page taller than the screen so the bottom buttons are hidden unless you scroll.